frappe/auth.py

```diff
@@ -460,9 +460,12 @@ def validate_ip_address(user):
 
 	user_info = frappe.get_cached_doc("User", user)
 	ip_list = user_info.get_restricted_ip_list()
+
 	if not ip_list:
 		return
 
+	# check if request ip is set before comaparing
+	check_request_ip()
```

Comment:
```suggestion
	check_request_ip()
```

The commend doesn't add anything of value. Name should be sufficiently self explanatory for 2 line function.